homeassistant/components/stream/worker.py

```diff
@@ -367,12 +367,14 @@ def flush(self, packet: av.Packet, last_part: bool) -> None:
                 data=self._memory_file.read(),
             ),
             (
-                segment_duration := float(
-                    (adjusted_dts - self._segment_start_dts) * packet.time_base
+                (
+                    segment_duration := float(
+                        (adjusted_dts - self._segment_start_dts) * packet.time_base
+                    )
                 )
-            )
-            if last_part
-            else 0,
+                if last_part
```

Comment:
This change doesn't seem related to f-strings.

Reply:
But ruff was automatically formatting it.